decidim-core/decidim-core.gemspec

```diff
@@ -57,5 +57,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency "decidim-participatory_processes", Decidim.version
```

Comment:
Should we add `decidim-assemblies` and/or `decidim-accountability` here?

Reply:
To be honest, `decidim-core` shouldn't depend on neither of them, and we should probably remove all the other dependencies as well, except for `decidim-dev`.

Reply:
Seems sensible.

Reply:
In reality, `decidim-core` might not work by itself. But it's a first step to detect unnecessary coupling.